nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/status/ProcessGroupStatusSnapshotDTO.java

```diff
@@ -81,6 +82,8 @@ public class ProcessGroupStatusSnapshotDTO implements Cloneable {
     private Integer terminatedThreadCount = 0;
     private Long processingNanos = 0L;
 
+    private ProcessingPerformanceStatus processingPerformanceStatus;
```

Comment:
Following the approach of other status classes, it looks like a new DTO class should be created and mapped so that it can be annotated.